server/app.ts

```diff
@@ -37,6 +38,7 @@ export default function createApp(services: Services): express.Application {
   app.use(nunjucksSetup(app, services))
   app.use(flash())
   app.use(setUpWebRequestParsing())
+  app.use(setUpMultipartFormDataParsing())
   app.use(setUpStaticResources())
```

Comment:
Could this middleware only operate on the file upload routes, rather than on every request? I think I did this on CVL, for uploading exclusion zone files, and limited the middleware only on that one route?

Reply:
You're right and the documentation explicitly warns against doing what I've done:

WARNING: Make sure that you always handle the files that a user uploads. Never add multer as a global middleware since a malicious user could upload files to a route that you didn't anticipate. Only use this function on routes where you are handling the uploaded files.

I've moved it to only being used on the upload prisoner list route and fixed the csrf problem by using a query string param